### PR TITLE
Feature/rangeslider page

### PR DESCRIPTION
**Pull request by PhungNg**
## Description

Implement the rangeslider page

## Motivation and Context

Dividing the different components in Forms page into seperate pages.

## How Has This Been Tested?

Ran and updated existing test, and added new snapshot

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/less/documentation-swedbankpay.less [hunk 1]
```diff
@@ -681,6 +681,20 @@
                 }
             }
 
+            &#no-tabs {
+                #undefined-ul {
```
Comment (VikingTristan) [inline src/less/documentation-swedbankpay.less:685]: Would you mind adding a comment above this css rule that explains why u need it?

**Review by VikingTristan — CHANGES_REQUESTED** [review]

File: src/less/documentation-swedbankpay.less (hunk 1 above)
Comment (PhungNg) [reply src/less/documentation-swedbankpay.less:685]: 🚀 🚀

File: src/App/components/FormComponents/Rangeslider.js [hunk 2]
```diff
@@ -12,30 +12,32 @@ const Rangeslider = ({ id, label, name, min, max, step, value, disabled, valueLa
...
+                </output>
+            </div>
+            <div className="value-range">{"\n"}
+                <small>{min}{valueLabelPostfix ? valueLabelPostfix : null}</small>{"\n"}
+                <small>{max}{valueLabelPostfix ? valueLabelPostfix : null}</small>{"\n"}
             </div>
-        </>
+        </div>
     );
 };
```
[the remark this answers is not in the document]
Comment (PhungNg) [reply src/App/components/FormComponents/Rangeslider.js]: 🚀

File: src/App/ComponentsDocumentation/components/Forms/index.js [hunk 3]
```diff
@@ -4,10 +4,11 @@ import { Link } from "react-router-dom";
 import { ComponentPreview, DocContainer, JavascriptDocs } from "@docutils";
 import InputGroup from "@components/InputGroup";
 import Button from "@components/Button";
-import { Checkbox, FormControlText, Radio, Rangeslider, Togglebox } from "@components/FormComponents";
+import { Checkbox, FormControlText, Radio, Togglebox } from "@components/FormComponents";
+import Rangeslider from "@components/Rangeslider";
```
Comment (VikingTristan) [inline src/App/ComponentsDocumentation/components/Forms/index.js:8]: This can be removed from this file now, no?

**Review by VikingTristan — APPROVED** [review]
Cool!
One final thing is that there is still a rangeslider on the forms page. Can it be deleted now that it has its own page?

**Review by VikingTristan — APPROVED** [review]
Looking good!

**Pull request merged**